community_supplied/Security/CGNAT/usf.cgnat.port-block-pool-info.rule

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
1. As a best-practice, rule name should not have dots hence change rule name with underscores or hypons
2. Rule syntax is wrong please fix it